opencodeblocks/scene/scene.py

```diff
@@ -158,6 +162,17 @@ def load_from_ipyg(self, filepath: str):
             data = json.loads(file.read())
         return data
 
+    def load_from_ipynb(self, filepath: str) -> OrderedDict:
```

Comment:
This function is the same as `load_from_ipyg`. Consider renaming `load_from_ipyg` to something like `load_from_json`.